Made minor changes to get_started_vertex_tensorboard

Tharun-SpringML opened the pull request:
Made minor changes to get_started_vertex_tensorboard file.
If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under the `# Community Notebooks` section, pointing to the author or the author's team.
- [x] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).

andrewferlitsch commented on notebooks/community/ml_ops/stage2/get_started_vertex_tensorboard.ipynb:
```diff
@@ -8,7 +8,7 @@
       },
```
<p>Need to add Open in Colab</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/437/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

andrewferlitsch replied:
<p>Where is the definition for {yes}? Or maybe I am missing something.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/437/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Tharun-SpringML commented:
Added Open in Colab text & logo in the notebook. Also added markup text for the code cell in which on executing , it needs to be provided an interactive yes/no input.
